charts/pathfinder-rpc-server/templates/blockchain-indexer-service.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert --controller statefulset -o pathfinder-rpc-server --volumes persistentVolumeClaim -c
+    kompose.version: 1.26.1 (a9d05d509)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: blockchain-indexer
```

Maybe we can rename this default Kompose label to a more common Kubernetes label often seen in this place, like `app:`?


```suggestion
    app: blockchain-indexer
```